includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -2796,6 +2796,7 @@ public static function render_single_url_list_table( $post ) {
 			<button type="button" class="button action keep"><?php esc_html_e( 'Keep', 'amp' ); ?></button>
 			<button type="button" class="button action reviewed-toggle reviewed"><?php esc_html_e( 'Mark reviewed', 'amp' ); ?></button>
 			<button type="button" class="button action reviewed-toggle unreviewed"><?php esc_html_e( 'Mark unreviewed', 'amp' ); ?></button>
+			<button type="button" class="button action copy-all"><?php esc_html_e( 'Copy error details to clipboard', 'amp' ); ?></button>
```

Comment:
I suggest making this consistent with the row action:

```suggestion
			<button type="button" class="button action copy-all"><?php esc_html_e( 'Copy to clipboard', 'amp' ); ?></button>
```

Reason is that when the button text changes from “Copy to clipboard” to “Copied to clipboard” it is then less of a horizontal shift than if it is from “Copy error details to clipboard” to “Copied to clipboard”. If that text is retained, then technically it should rather have the text “Copied error details to clipboard”.

The shorter text also means less horizontal width for a space that is already getting cramped.

Reply:
I like this idea.